src/VerifyU2FController.js

```diff
@@ -46,8 +48,16 @@ function (Okta, FormController, FormType, FooterSignout, Q) {
             var deferred = Q.defer();
             u2f.sign(appId, factorData.challenge.nonce, registeredKeys, function (data) {
               self.trigger('errors:clear');
+              var errorSummary = 'Error Code: ' + data.errorCode;
               if (data.errorCode && data.errorCode !== 0) {
-                deferred.reject({ responseJSON: {errorSummary: 'Error Code: ' + data.errorCode}});
+                switch (data.errorCode) {
+                case 4:
```

Comment:
It actually can
From ybukey's website:

> The presented device is not eligible for this request. For a registration request this may mean that the token is already registered, and for a sign request it may mean that the token does not know the presented key handle.